core/domain/question_jobs_one_off.py

```diff
@@ -86,5 +76,3 @@ def reduce(key, values):
         elif key == QuestionMigrationOneOffJob._MIGRATED_KEY:
             yield (key, ['%d questions successfully migrated.' % (
                 sum(ast.literal_eval(v) for v in values))])
-        else:
```

Comment:
Are you sure this branch will never be hit in production?

Reply:
@lilithxxx you shouldn't delete this, just as you shouldn't delete the code above. Please find a way to test it.

Reply:
Done